### DISCUSSION TITLE
etc/giscus/

### DISCUSSION THREAD

**Discussion by giscus** [chlwlstlf/chlwlstlf.github.io#3] [Announcements]
# etc/giscus/

giscus로 댓글 기능 추가하기

https://chlwlstlf.github.io/etc/giscus/

**Comment by chysis** [comment]
GitHub와 연동해서 쉽게 댓글을 남길 수 있게 되었네요!

**Reply by chlwlstlf** [reply]
이게 훨씬 편한 것 같아요!!!!!!!!! 감사합니다🤗

**Comment by pakxe** [comment]
wow. 당신의 열렬한 fan이었는데 이렇게 comment를 남길 수 있어 너무나도 happy예요 XD

**Reply by chlwlstlf** [reply]
외국인이신가요???? my blog 봐줘서 so thank you예요🙇‍♀️

**Comment by rbgksqkr** [comment]
덕분에(?) 댓글 설정했어요 감사합니다 😁

https://rbgksqkr.github.io/

**Reply by chlwlstlf** [reply]
본인 블로그 홍보.?!??!? 이미 댓글 달았습니다. 찾아보시죠.
